tests/test_project_structure.py

```diff
@@ -43,7 +43,6 @@
     'tests/providers/jenkins/hooks/test_jenkins.py',
     'tests/providers/microsoft/azure/sensors/test_azure_cosmos.py',
     'tests/providers/microsoft/mssql/hooks/test_mssql.py',
-    'tests/providers/qubole/hooks/test_qubole.py',
```

Comment:
why this file has been removed?

Reply:
Because we have added the test_qubole.py by [this commit](https://github.com/apache/airflow/pull/9349/commits/d48ea0a167c4428c37fc2939779f11d03fef9863), and when the CI run I received a CI message to remove that line.

Pls cmiiw.

Reply:
let me check. There might be a problem when merging master.

Reply:
This is a test that checks for missing test files. This file has been added to this change, so we need to update this list to represent the current state.  

This test has been added to inform the user about the project rules as soon as possible. No reviewer needs to inform users that we need unit tests.

It also simplifies the work of project maintainer, because it provides easy insight into project defects, which allows engaging other contributors to these tasks. More info: https://github.com/apache/airflow/issues/8278

Reply:
This is the reason why we remove the line:
![image](https://user-images.githubusercontent.com/39547572/85279432-98431b80-b4b0-11ea-859b-acdd29f34b87.png)

https://github.com/apache/airflow/pull/9349/checks?check_run_id=781002396

I hope this clears up the issue.

Reply:
@msumit can I resolve this?